docs/commands/dev.md

```diff
@@ -24,7 +24,7 @@ netlify dev
 - `framework` (*string*) - framework to use. Defaults to #auto which automatically detects a framework
 - `functions` (*string*) - specify a functions folder to serve
 - `functionsPort` (*string*) - port of functions server
-- `geo` (*cache | mock | update*) - force geolocation data to be updated, use cached data from the last 24h if found, or use a mock location
+- `geo` (*string*) - force geolocation data to be updated, use cached data from the last 24h if found, or use a mock location
```

Comment:
Could be helpful to add an example of using a custom mock location so that folks don't have to trigger the error message / spelunk the CLI code to figure out what format to use. 

FYI, I filed an issue for mentioning this new feature in docs.netlify.com at https://github.com/netlify/docs/issues/2169

Reply:
On this note, it's probably best to keep `geo` as an enum with `cache | mock | update` and introduce a new flag (e.g. `country`) to be used with `mock`.

So:

```
# Uses the default mock location
ntl dev --geo=mock

# Uses a custom mock location
ntl dev --geo=mock --country=CA
```